java/src/test/java/com/google/appengine/tools/mapreduce/outputs/SizeSegmentedGoogleCloudStorageFileOutputTest.java

```diff
@@ -36,14 +37,15 @@ public class SizeSegmentedGoogleCloudStorageFileOutputTest {
 
   GoogleCloudStorageFileOutput.Options options;
 
+  @Getter
+  String bucket;
+  @Getter @Setter(onMethod_ = @BeforeEach)
```

Comment:
is the `(onMethod_ = @BeforeEach)` really needed? 
Shouldn't it be injected in the extension in the same way the bucket is done?

Reply:
this is using a junit ParameterResolver to do it.

The `String bucket` thing is done with reflection by our own class